projects/observability/src/pages/explorer/explorer.component.ts

```diff
@@ -339,3 +379,8 @@ const enum ExplorerQueryParam {
   GroupLimit = 'limit',
   Series = 'series'
 }
+
+export interface SavedQuery {
+  scope: ScopeQueryParam;
+  filters: string[];
```

Comment:
I think `filters` here is not the ideal word, maybe something like `filterUrlStrings`? Given we are only storing the `urlString` for filters.

Reply:
This field seems to map 1-to-1 to the `filter` parameter required by the `navigate` method in `NavigationService`. I'm using it like this in the follow-up PR:

<img width="304" alt="Screenshot 2022-05-16 at 11 56 22 AM" src="https://user-images.githubusercontent.com/29686866/168532037-aff1dbc5-c566-45eb-877c-93441ff95f57.png">

But I agree, the naming shouldn't raise any confusion. Renaming it to `filterUrlStrings`.
Usage will change to `filter: query.filterUrlStrings` in the above screenshot.

Reply:
Done ✅